Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -495,7 +495,7 @@ static Vector<UserContentURLPattern> parseAndAllowAccessToCORSDisablingPatterns(
     return parsedPatterns;
 }
 
-static std::variant<UniqueRef<FrameLoaderClient>, UniqueRef<RemoteFrameClient>> clientForMainFrame(Ref<WebFrame>&& mainFrame, bool local)
+static std::variant<UniqueRef<FrameLoaderClient>, PageConfiguration::RemoteMainFrameCreationParameters> clientForMainFrame(Ref<WebFrame>&& mainFrame, bool local, std::optional<WebCore::ProcessIdentifier> remoteProcessIdentifier)
```

Comment:
We don't need the local boolean any more.  It is redundant info with the std::optional, which contains nullopt for local main frames and a process identifier for remote frames.

Reply:
good point